model.py

```diff
@@ -195,7 +197,7 @@ def __init__(self, config: ModelArgs) -> None:
         self.w2 = nn.Linear(config.intermediate_size, config.dim, bias=False)
 
     def forward(self, x: Tensor) -> Tensor:
-        return self.w2(F.silu(self.w1(x)) * self.w3(x))
+        return self.w2(F.gelu(self.w1(x)) * self.w3(x))
```

Comment:
This should be the approximate gelu instead https://x.com/danielhanchen/status/1763743097417314403?s=20
```suggestion
        return self.w2(F.gelu(self.w1(x), approximate="tanh") * self.w3(x))
```